tests/EvenNumber.t.sol

```diff
@@ -34,21 +34,26 @@ contract EvenNumberTest is BonsaiTest {
         assertEq(evenNumber.get(), 0);
     }
 
-    function test_Set_Even() public {
-        set(12345678);
-    }
+    function test_SetEven() public {
+        uint256 number = 12345678;
+        (bytes memory journal, bytes32 post_state_digest, bytes memory seal) =
+            queryImageOutputAndSeal(imageId, abi.encode(number));
 
-    function testFail_Set_Odd() public {
-        set(123456789);
+        evenNumber.set(abi.decode(journal, (uint256)), post_state_digest, seal);
+        assertEq(evenNumber.get(), number);
     }
 
-    function test_Set_Zero() public {
-        set(0);
+    function testFail_SetOdd() public {
+        uint256 number = 123456789;
+        // Attempting to prove that 123456789 is even should fail.
+        queryImageOutputAndSeal(imageId, abi.encode(number));
     }
 
-    function set(uint256 number) internal {
```

Comment:
Why removing this? I think the way it was before looked cleaner

Reply:
With test code, my style preference if general in favor of a bit more duplication if it means each test is more self contained. This also makes it more clear where the failure case is actually failing, instead of including additional lines of code that should never be reached.